src/js/big-play-button.js

```diff
@@ -8,16 +11,20 @@
  * @class
  * @constructor
  */
-vjs.BigPlayButton = vjs.Button.extend();
+var BigPlayButton = Button.extend();
```

Comment:
See this commit for a list of the jshint reverts https://github.com/mmcc/video.js/commit/f82e4d93105b05292950cf1f8696cb8e8764998a